src/Admin.php

```diff
@@ -35,6 +35,10 @@ public static function init() {
     // Exposes SVG images in media library.
     add_filter('wp_prepare_attachment_for_js', __CLASS__ . '::wp_prepare_attachment_for_js');
     add_action('admin_head', __CLASS__ . '::admin_head');
+
+    // Excludes users with subscriber role from the author dropdown select list to avoid
+    // an extremely long list in the the case of sites with a large amount of subscribers.
```

Comment:
The large list is not really the problem – the problem is the page loading performance:
```diff
-    // Excludes users with subscriber role from the author dropdown select list to avoid
-    // an extremely long list in the the case of sites with a large amount of subscribers.
+    // Exclude subscribers from post author select options to prevent a performance
+    // slowdown on sites with large amounts of non-administrative registered users.
```